src/components/map/HfpLayer.js

```diff
@@ -65,22 +69,67 @@ Speed: ${hfpItem.spd} km/h`;
     }
   };
 
+  componentWillUnmount() {
+    console.log("unmount hfp line");
+  }
+
   render() {
     const {name} = this.props;
-    const color = getColor(name);
+
+    // Split the positions into chunks by the delay.
```

Comment:
For some reason this sometimes leaves empty gaps between chunks. Couple of those was quite long.